pagopa/io-app-design-system
# [IOPLT-135] Refinement of the `HeaderSecondLevel` Component

## Short description
This PR refines the component definition of the Header of the second level navigation pages.

## How to test
Check the related Header Second Level example app page to verify everything's fine
